Lesson11_Task01.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
(venv) [antoha@dlhn test_project]$ ./bin/lint ./T-HOMETASKS/L11/l11_h01_MaximFedoseev.py
./T-HOMETASKS/L11/l11_h01_MaximFedoseev.py:1:1: D205 1 blank line required between summary line and description
./T-HOMETASKS/L11/l11_h01_MaximFedoseev.py:1:1: D400 First line should end with a period
./T-HOMETASKS/L11/l11_h01_MaximFedoseev.py:8:1: E800 Found commented out code
./T-HOMETASKS/L11/l11_h01_MaximFedoseev.py:9:1: E800 Found commented out code
./T-HOMETASKS/L11/l11_h01_MaximFedoseev.py:10:1: E800 Found commented out code
./T-HOMETASKS/L11/l11_h01_MaximFedoseev.py:11:1: E800 Found commented out code
./T-HOMETASKS/L11/l11_h01_MaximFedoseev.py:16:1: D103 Missing docstring in public function
./T-HOMETASKS/L11/l11_h01_MaximFedoseev.py:18:5: E731 do not assign a lambda expression, use a def
./T-HOMETASKS/L11/l11_h01_MaximFedoseev.py:36:16: Q000 Double quotes found but single quotes preferred